DelayedClipboard/DelayedClipboardRenderingExplainer.md

```diff
@@ -75,9 +73,7 @@ const clipboardItemInput = new ClipboardItem({['image/png']: blobInput1, ['text/
 navigator.clipboard.write([clipboardItemInput]);
 ```
 
-The disadvantage of the latter approach is that it adds overhead to the web author, as it is a new function for them to learn. However, as the clipboard item constructor remains the same, web authors that want to adopt delayed clipboard rendering in their existing web applications will just be able to move the generation of data to the callbacks map and pass it to addDelayedWriteCallback, without needing to change their existing clipboard item’s constructor.
-
-If the web author doesn't provide a delayed rendering callback or data, then an error is thrown in the source application and the write operation fails.
+The disadvantage of the latter approach is that it adds overhead to the web author, as it is a new function for them to learn. **As a result, we think that adding a new argument to the `ClipboardItem` constructor is the preferred solution.**
```

Comment:
Nit: Consider rephrasing it as below:
"As a result, we are choosing the solution described in <link to the proposed solution>"